### ISSUE TITLE
Create a new plan

### ISSUE THREAD

**Issue by ivanminutillo** [opencooperativeecosystem/kanban#14]
Currently the procedure to create detailed plan with processes and commitments is a bit cumberstone. If we would give the possibility to create plans with multiple processes easily in the new kanban app, I think it would be a great achievement and could speed up the adoption / testing of the app from our user base.

**Comment by fosterlynn** [comment]
Yes, it could include planning from recipe and planning from scratch.  Having a recipe makes the planning very easy.  But both planning from scratch and creating a recipe are cumbersome in the OCP django app.  It would be great to figure out a new and better UX/UI to build these.  (I think they will be similar, but I'll start a new issue for recipes.)

**Comment by bhaugen** [comment]
I'd love to rethink and redevelop recipes with a good UI. The one in django works, but it is clumsy and hacky.

I don't know how you would make it mobile-friendly, though. I think it needs a nice tree UI with more canvas than you can get on a phone. But my imagination might be limited.
